_includes/community/community-contributions-band.html

```diff
@@ -0,0 +1,56 @@
+<div class="component-wrapper contributions-band">
+  <div class="width-3-12 width-12-12-m contrib-block">
```

Comment:
Presumably these are all supposed to be links?

Reply:
No problem. Do these seem like the correct links to you? 

Download Strimzi: https://deploy-preview-130--strimzi.netlify.com/downloads
Follow Strimzi on Twitter: https://twitter.com/strimziio
Submit a bug: https://github.com/strimzi/strimzi-kafka-operator/issues
Submit some code: https://github.com/strimzi/strimzi-kafka-operator/labels/good-start
Join our chatroom: https://slack.cncf.io/
Edit the Documentation: https://deploy-preview-130--strimzi.netlify.com/contributing/guide/
Blog with us: https://github.com/strimzi/strimzi.github.io
Translate into your language: https://deploy-preview-130--strimzi.netlify.com/contributing
Spread the word: https://twitter.com/strimziio

They're just my best guesses :)

Reply:
We obviously need the links to not use the `https://deploy-preview-130--strimzi.netlify.com` part, but I guess that is clear. Apart frm that I think these would be a good start.